docs/blog/2023-Q3-community-board.rst

```diff
@@ -0,0 +1,58 @@
+.. post:: Aug 3, 2023
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q3-2023
+=======================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the third quarterly update for 2023. 
+
+To catch up on the previous quarterly updates, check out `Q1 blog post <https://www.writethedocs.org/blog/2023-Q1-community-board/>`_ and `Q2 blog post <https://www.writethedocs.org/blog/2023-Q2-community-board/>`_. 
+
+Q3 quarterly meeting highlights
+-------------------------------
+
+Our third meeting covered several topics: 
+
+**Meetup organizer mailing list and survey**
+    Earlier this year, our Meetups team lead Rose Williams created a mailing list of all the meetup organizers, in an effort to formalize communication channels so that we can support the various meetups around the world more easily. Rose also sent out a survey to all the meetup organizers where they can indicate whether their meetup is still active, and submit ideas on how to evolve local and regional meetup structure. 
```

Comment:
I think this is great, I think this might be a more clear explanation. Let me know if you agree!

"This year the Meetup team is focusing on updating organizer resources and better communication. To support our efforts the Meetup team lead, Rose Williams, created an organizer mailing list and distributed a meetup activity survey. The survey assessed meetup activity and collected ideas for evolving the local and regional meetup structure."

Reply:
I love it! Thanks for the suggestion.